src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -32,6 +32,8 @@
     <Compile Include="ParseTestCoverageInfo.cs" />
     <Compile Include="RemoveDuplicatesWithLastOneWinsPolicy.cs" />
     <Compile Include="PrereleaseResolveNuGetPackageAssets.cs" />
+    <Compile Include="UpdateProjectDependencies.cs" />
+    <Compile Include="ValidateProjectDependencyVersions.cs" />
```

Comment:
Added these to the net45 project, added the NuGet.Versioning dependency to its project.json, and fixed up other conflicts. It still builds and works for me (although I haven't tested the net45 version).